cluster/storage/src/main/java/com/linkedin/openhouse/cluster/storage/selector/impl/StorageInNameRegexSelector.java

```diff
@@ -0,0 +1,74 @@
+package com.linkedin.openhouse.cluster.storage.selector.impl;
+
+import com.google.common.base.Preconditions;
+import com.linkedin.openhouse.cluster.storage.Storage;
+import com.linkedin.openhouse.cluster.storage.StorageManager;
+import com.linkedin.openhouse.cluster.storage.StorageType;
+import com.linkedin.openhouse.cluster.storage.configs.StorageProperties;
+import com.linkedin.openhouse.cluster.storage.selector.BaseStorageSelector;
+import com.linkedin.openhouse.cluster.storage.selector.StorageSelector;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import javax.annotation.PostConstruct;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+/**
+ * An implementation of {@link StorageSelector} that takes a regex and storage-type provided in the
+ * storage selector params in the yaml configuration and returns storage that's provided if the db
+ * and table name matches the regex, else returns the cluster storage default
+ */
```

Comment:
can you add an example of configuration file looks like for setting this up.. 
ex:
```
storages:
   storage-selector:
          a:
          b:
          c:
```

Reply:
done